crates/apub/src/activities/community/report.rs

```diff
@@ -50,8 +50,11 @@ impl Report {
       id: id.clone(),
       audience: Some(community.id().into()),
     };
-    // todo: this should probably filter and only send if the community is remote?
-    let inbox = ActivitySendTargets::to_inbox(community.shared_inbox_or_inbox());
+    let inbox = if community.local {
+      ActivitySendTargets::empty()
+    } else {
+      ActivitySendTargets::to_inbox(community.shared_inbox_or_inbox())
+    };
     send_lemmy_activity(&context, report, &actor, inbox, false).await
```

Comment:
I believe these should all use `send_activity_in_community()` (same for follow.rs and undo_follow.rs). Not sure why its not the case now.

Reply:
I was wondering that too and it would need a bit of modifications because send_activity_in_community has some other things that trigger that don't trigger here (send to person followers, announce locally)